modules/opensearch/opensearch.client.inc

```diff
@@ -376,7 +376,7 @@ function opensearch_execute_cache($request) {
 
   // Handle fulltext vs. dkabm caching of object.
   $type = OPENSEARCH_CACHE_TING_OBJECT;
-  if ($parms['format'] == 'docbook') {
+  if ($parms['objectFormat'] == 'docbook') {
```

Comment:
This change is not mentioned in commit or PR message. I thought this was an unrelated change. The only way I can tell this is needed is because I reread the Redmine issue.

Please remember to document this here as well.